fdbserver/worker.actor.cpp

```diff
@@ -880,17 +956,17 @@ ACTOR Future<Void> healthMonitor(Reference<AsyncVar<Optional<ClusterControllerFu
 			const auto& allPeers = FlowTransport::transport().getAllPeers();
 			UpdateWorkerHealthRequest req;
 
-			bool workerInDb = false;
-			bool workerInPrimary = false;
+			enum WorkerLocation { None, Primary, PrimarySatellite, Remote };
```

Comment:
So PrimarySatellite is just one enum value, but never used?

Reply:
Good catch. Yes, I was deciding between whether we want worker in satellite to monitor primary or not. Later I decided not to do so for now. Removed PrimarySatellite.